entries/jQuery.validator.methods.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="jQuery.validator.methods" returns="Object">
+	<title>jQuery.validator.methods</title>
+	<desc>Object holding all validation methods known to the validator</desc>
```

Comment:
Amend: "This can be accessed to override individual methods, while keeping the default messages."